src/app/devices/deviceList/components/deviceQueryClause.tsx

```diff
@@ -65,7 +65,7 @@ export default class DeviceQueryClause extends React.PureComponent<DeviceQueryCl
     }
 
     private readonly shouldShowOperator = (parameterType: ParameterType) => {
-        return ParameterType.capabilityModelId !== parameterType && ParameterType.interfaceId !== parameterType;
+        return parameterType && ParameterType.capabilityModelId !== parameterType && ParameterType.interfaceId !== parameterType;
```

Comment:
Seems redundant. Was there a case where the existing comparisons didn't work as expected?

Reply:
The case when paremeterType is null is not covered. So three input field is renders by default, where showing two is what I have in mind as expected, because the first option (DCM) from the list renders two inputs so it feels like slightly less disruptive for the users.